documentation/modules/security/con-securing-client-authentication.adoc

```diff
@@ -82,6 +76,31 @@ data:
   user.password: # Password for protecting the PKCS #12 archive file
 ----
 
+When you configure a client, you specify the following:
+
+* *Truststore* properties for the public cluster CA certificate to authenticate with the Kafka cluster
```

Comment:
Maybe this would be more clear to regular users

```suggestion
* *Truststore* properties for the public cluster CA certificate to verify the identity of the Kafka cluster
```